addons/mrp/i18n/mrp.pot

```diff
@@ -233,7 +233,7 @@ msgstr ""
 
 #. module: mrp
 #: model_terms:ir.ui.view,arch_db:mrp.mrp_production_form_view
-msgid "<span class=\"text-bf\">To Produce</span>"
+msgid "<span class=\"font-weight-bold\">To Produce</span>"
```

Comment:
I bet this is not really required (we don't change translations in master), probably a grep going a bit too far ?